packages/@sanity/components/src/toggles/styles/Checkbox.css

```diff
@@ -36,6 +36,10 @@
   composes: root;
 }
 
+.undefinedChecked {
```

Comment:
Would it be better to call it `.indeterminate`?

Reply:
👍